### PR TITLE
Fixed Program Timeline Sections of The All Past ScholarX Programs

### LINKED ISSUE #1416: Fix Program Timeline section in past scholarx programs
**Describe the bug**
Issue when you view past scholarX program section using smartphones

**To Reproduce**
1. Go to 'ScholarX page past programs'
2. View Timeline section using smartphones
3. See error

**Expected behavior**
Able to view timeline section using smartphones

**Screenshots**
Screenshot 2022-11-22 011307

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by janithlahirukariyawasam**
## Purpose

The purpose of this PR is to fix #1416 

## Goals

With this fixing, viewers can see all past timeline sections without facing any issues.

## Approach

Used HTML tables.

Fixed,
 **2021 Timeline**
 **2020 Timeline**

With this, **all past program timeline** sections are up to date.

### Screenshots

**ERROR**
image

error pull2
**After Fixing**
image

fix pull 2

  
### Preview Link

https://pr-1417-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/past-programs/2020/styles.css [hunk 1]
```diff
@@ -79,6 +79,14 @@
...
 }
 
+.program-timeline-row{
+    width: 100%;
+}
+
+.program-timeline-date{
+    text-align: end !important;
+}
+
```
Comment (anjula-sack) [inline scholarx/past-programs/2020/styles.css]: Let's move this to` /assets/css/global-custom.css` so we can avoid duplicating the same code

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by YoshithaRathnayake** [comment]
@anjula-sack is it ok if I approve this bcs there are no changes requested no?

File: assets/css/global-custom.css [hunk 2]
```diff
@@ -740,3 +740,11 @@ li.timeline-element:last-child {
...
     }
 }
+
+.program-timeline-row{
+    width: 100%;
+}
+
+.program-timeline-date{
+    text-align: end !important;
+}
```
Comment (YoshithaRathnayake) [inline assets/css/global-custom.css]: Add end of the line @janithlahirukariyawasam

**Review by YoshithaRathnayake — DISMISSED** [review]

File: assets/css/global-custom.css (hunk 2 above)
Comment (janithlahirukariyawasam) [reply assets/css/global-custom.css]: @YoshithaRathnayake Can you check now?

**Review by anjula-sack — APPROVED** [review]
Looks great! Thank you! @janithlahirukariyawasam

**Pull request merged**